pkg/client/pipelines.go

```diff
@@ -132,7 +133,8 @@ func JobSpecFromJobSpecFragment(gateName string, jsFragment *console.JobSpecFrag
 		gateNameAnnotationKey := v1alpha1.GroupVersion.Group + "/gatename"
 		jobSpec.Template.ObjectMeta.Annotations[gateNameAnnotationKey] = gateName
 	}
-	
+	jobSpec.TTLSecondsAfterFinished = lo.ToPtr(int32(86400))
```

Comment:
could we make the int a const with a descriptive name so its less of a magic number

Reply:
done